docs/readthedocs/source/doc/PPML/Overview/examples.rst

```diff
@@ -3,6 +3,8 @@ Tutorials & Examples
 
 * `A Hello World Example <../Overview/quicktour.html>`__ is a very simple exmaple for getting started.
 
-* `PPML e2e Example <../QuickStart/end-to-end.html>`__ introduces the end-to-end PPML workflow using SimpleQuery as an example.
+* `PPML E2E Example <../QuickStart/end-to-end.html>`__ introduces the end-to-end PPML workflow using SimpleQuery as an example.
+
+* `PPML E2E on Azure <../Overview/azure_ppml.html>`__ introduces the end-to-end PPML workflow on Azure Cloud.
```

Comment:
* `PPML E2E Example <../QuickStart/end-to-end.html>`__ introduces the end-to-end PPML workflow using SimpleQuery as an example.

* `PPML E2E Example on Azure <../Overview/azure_ppml.html>`__ introduces the end-to-end PPML workflow on Azure Cloud using xxx as an example.